var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -51,7 +51,7 @@ class Paraview(CMakePackage, CudaPackage):
             description='Builds a shared version of the library')
     variant('kits', default=True,
             description='Use module kits')
-    variant('cuda_arch', default='native', multi=False,
+    variant('cuda_arch_family', default='native', multi=False,
```

Comment:
It is definitely preferable to add a new variant than to override an inherited one with different values (i.e., going from an 'integer' to 'text.

Reply:
Yes, here we are adding a new variant instead of overriding `cuda_arch`